NEWS.md

```diff
@@ -2,9 +2,43 @@
 
 ## v11 (unreleased)
 
+- Support for Rocky Linux, Alma Linux, and Gentoo has been added!
+- A new `ManifestFormat=` option can be used to generate "manifest" files that
+  describe what packages were installed. With `json`, a JSON file that shows
+  the names and versions of all installed packages will be created. With
+  `changelog`, a longer human-readable file that shows package descriptions and
+  changelogs will be generated. This latter format should be considered
+  experimental and likely to change in later versions.
+- A new `RemovePackages=` option can be used to uninstall packages after the
+  build and finalize scripts have been done. This is useful for the case where
+  packages are required by the build scripts, or pulled in as dependencies
+  for scriptlets of other packages, but are not necessary in the final image.
+- A new `BaseImage=` option can be used to build "system extensions" a.k.a.
+  "sysexts" — partial images which are mounted on top of an existing system
+  to provide additional files under `/usr/`.
```

Comment:
maybe add link to https://www.freedesktop.org/software/systemd/man/systemd-sysext.html